service/src/test/java/uk/nhs/adaptors/gp2gp/common/service/ConfidentialityServiceTest.java

```diff
@@ -0,0 +1,92 @@
+package uk.nhs.adaptors.gp2gp.common.service;
+
+import org.hl7.fhir.dstu3.model.Coding;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.nhs.adaptors.gp2gp.common.configuration.RedactionsContext;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(MockitoExtension.class)
+public class ConfidentialityServiceTest {
+    private static final List<Coding> NO_META_SECURITY = List.of(new Coding());
+    private static final List<Coding> NOPAT_META_SECURITY = List.of(
+        new Coding(
+            "http://hl7.org/fhir/v3/ActCode",
+            "NOPAT",
+            "no disclosure to patient, family or caregivers without attending provider's authorization"
+        )
+    );
```

Comment:
Rather than `list.of()`, we could use `Collections.singletonList()` here 😄

Reply:
Hmm is there a particular reason for `Collections.SingletonList()`, most places I've read seem to thing that they are pretty much the same, but you should use ``List.of() as it is a newer language feature, such as below:

"_If you can use a more recent Java version, it's preferable for its familiarity, clarity and because we use List interface much more than Collections. If restricted to an older Java version, Collections.singletonList remains a solid choice._"

"_Both Collections::singletonList and List:of are great choices for creating single-element lists. If you're fortunate enough to be using a version of Java that supports both methods (9 and above), then I recommend going with List:of for its ease of use, readability, and better-documented immutability._"

Reply:
I'll go with it because it make little odds and I'm making the other changes anyway!